src/reset.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>reset</title>
+</head>
+<body>
+    <h1>monofile password reset</h1>
+    <form>
+        <input name="password" type="password">
```

Comment:
```suggestion
        <input name="password" type="password" autocomplete="new-password">
```

https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/autocomplete#sect13
> A new password. When creating a new account or changing passwords, this should be used for an "Enter your new password" or "Confirm new password" field, as opposed to a general "Enter your current password" field that might be present. This may be used by the browser both to avoid accidentally filling in an existing password and to offer assistance in creating a secure password.